Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.9
+// swift-tools-version: 6.0
```

Comment:
This would break existing users on pre-6.0 toolchains, so we can't do it this way.

Reply:
I would propose to have two `Package.swift` files, one generic and one specific for Swift 6.0 (`Package@swift-6.0.swift`)

Reply:
We likely won't adopt any features that require Swift 6 while we still support Swift 5, so the first goal here would be to fix as many warnings as possible that come up with `-Xswiftc -strict-concurrency=complete`.